### ISSUE TITLE
Disable when auto move to next git revert changes

### ISSUE THREAD

**Issue by mifas** [Microsoft/vscode#40255]
- VSCode Version: Code 1.19.0 (816be6780ca8bd0ab80314e11478c48c70d09383, 2017-12-14T12:06:43.492Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-intelephense-client|bme|0.8.2
sublime-keybindings|ms-|3.0.3
vscode-docker|Pet|0.0.22
material-icon-theme|PKi|3.1.0

(1 theme extensions excluded)

---

Steps to Reproduce:

1. In the git repo, made some changes
2. when you click the gutter and make revert changes, it'll auto go to next changes

[DEMO](https://vimeo.com/247432763)

This is annoying and we lose our focus on the document. 

Also below shortcut is not working. We need to manually click the button

```
{
  "key": "ctrl+alt+z",
  "command": "git.revertChange"
}
```
Basically it's should work as PHPStorm.

Reproduces without extensions: Yes/No

**Comment by joaomoreno** [comment]
#39307

**Comment by mifas** [comment]
Thanks, Also be note, shortcut keys are not working,
